packages/components/src/Text/useText.js

```diff
@@ -128,7 +132,16 @@ export function useText(props) {
 	 */
 	if (!truncate && is.array(children)) {
 		content = React.Children.map(children, (child) => {
-			if (!is.plainObject(child)) return child;
+			// Thiis is... unfortunate. Would we rather throw a `ts-ignore` here than increase the complexity of this loop?
```

Comment:
/cc @ItsJonQ @sirreal

Reply:
I mean. I'm okay with that 😂

Reply:
Yeah I think I'll just add some surgical ts-ignores. Otherwise this is too complex for a loop